src/js/api/urls.js

```diff
@@ -30,6 +30,9 @@ export const STOCK_MOVEMENT_ITEM_REMOVE = id => `${STOCK_MOVEMENT_ITEM_API}/${id
 // STOCK TRANSFER
 export const STOCK_TRANSFER_API = `${API}/stockTransfers`;
 export const STOCK_TRANSFER_DELETE = id => `${STOCK_TRANSFER_API}/${id}`;
```

Comment:
yeah, I think you've followed what was there, but I think this should also be `_BY_ID`. (Like for STOCK_MOVEMENT)

Reply:
or perhaps just `STOCK_TRANSFER` without `_BY_ID` :)

Reply:
basically, it would be better to name it as it was done for stock movements
```
export const STOCK_TRANSFER_API = `${API}/stockTransfers`;
export const STOCK_TRANSFER_BY_ID = id => `${STOCK_TRANSFER_API}/${id}`;
```
it couldn't be just `STOCK_TRANSFER`, because we have to separate the URLs for sending requests and redirects.

Reply:
Oh, that's why it was done like that, forgot about redirects :/